javers-core/src/main/java/org/javers/snapshots-compiler/JaversSnapshotsCompiler.java

```diff
@@ -0,0 +1,216 @@
+package org.javers;
```

Comment:
there is no tests here, see how we write tests in Javers and add some
https://github.com/javers/javers/blob/master/javers-core/src/test/groovy/org/javers/core/JaversDiffE2ETest.groovy